### DISCUSSION TITLE
[Feature Request] add more info to vscode help text for out of date Mojo SDK

### DISCUSSION THREAD

**Discussion by hirehamir** [modular/modular#2003] [Ideas]
The pop-up modal for an out of date SDK in the VSCode extension should contain a link to update instructions.

It'll save developers time if they have a direct link to update instructions in the pop-up modal.

```
The current Mojo SDK version is incompatible with this version of the Mojo extension.
Please update your SDK to ensure the extension behaves correctly.
```
![image](https://github.com/modularml/mojo/assets/43425812/a2321327-68f0-4a47-baf2-dc12644fcaf0)

**Comment by tengo-group** [comment]
I have the same proble. How shell we deal with this?

**Comment by hogepodge** [comment]
We released new versions of Mojo and MAX last week. If your VSCode plugin updates and is out of sync with the version installed on your system, it will throw this error. You can update your installation using the modular command:

`modular update mojo` if you're just updating the Mojo package, and `modular update max` if you're on Linux and are updating MAX along with Mojo.
